packages/snjs/lib/services/item_manager.ts

```diff
@@ -721,6 +718,29 @@ export class ItemManager extends PureService {
     return searchArray(this.tags, { title: title });
   }
 
+  /**
+   * Finds tags with titles starting with a search query
+   */
+  public searchTags(searchQuery: string): SNTag[] {
+    const delimiter = '.';
+    return this.tags
+      .filter((tag) =>
+        tag.title
+          .toLowerCase()
+          .split(delimiter)
+          .some((component) => component.startsWith(searchQuery.toLowerCase()))
+      )
+      .sort(
+        typeof Intl !== 'undefined'
+          ? (a, b) =>
+              new Intl.Collator('en', { numeric: true }).compare(
```

Comment:
I read before I believe that Collator objects should be reused if possible?

Reply:
Collator should also be safe to use at this point 🙂 https://caniuse.com/?search=collator

Reply:
It's safe for pretty much all browsers but Hermes doesn't have support for Intl yet, so we need the undefined check for mobile